validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -182,10 +185,15 @@ public void onBlockImported(ImportedBlockEvent event) {
   @Subscribe
   public void onAttestationEvent(BroadcastAttestationEvent event) throws IllegalArgumentException {
     try {
+
+      UnsignedLong slot = event.getNodeSlot();
+      if (slot.compareTo(latestBroadcastAttestationEventSlot) <= 0) {
+        return;
+      }
```

Comment:
I guess that resolves my concerns about getting slashed above, but this still feels very weird to me.